tubemq-server/src/test/resource/data/00000000000000000000.tube

```diff
@@ -0,0 +1 @@
+abcabc
```

Comment:
Does the test data need to be combined?

Reply:
sorry, it's a wrong file, i will delete it.